README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/andela/ah-backend-odin.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-odin)
```

Comment:
The travis badge is currently pointing to the develop branch, which i think is okay.  My question is if the master(stable branch) should be displaying the develop build's badge.